src/java/edu/slu/tpen/servlet/ProjectServlet.java

```diff
@@ -85,15 +85,17 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                 //System.out.println("Project 1");
                 String check = "transcribe";
                 String redirect = req.getPathInfo().substring(1);
-                if (redirect.contains(check)) {
-                    projID = parseInt(redirect.replace("/" + check, ""));
-                    String redirectURL = req.getContextPath() + "/transcription.html?projectID=" + projID;
-                    resp.sendRedirect(redirectURL);
-                } else {
+                int projInt = redirect.length();
+                if( projInt > 0) {
+                    if (redirect.contains(check)) {
+                        projID = parseInt(redirect.replace("/" + check, ""));
+                        String redirectURL = req.getContextPath() + "/transcription.html?projectID=" + projID;
+                        resp.sendRedirect(redirectURL);
+                }   else {
                     //System.out.println("Project 2");
                     projID = parseInt(req.getPathInfo().substring(1).replace("/", "").replace("manifest.json",""));
                     Project proj = new Project(projID);
-                    //System.out.println("Project 3");
+                    System.out.println("Project 3");
```

Comment:
Clean these commented and active System.out's, we would rather not deploy superfluous logs